contrib/babelfishpg_tsql/src/backend_parser/parser.c

```diff
@@ -29,8 +29,6 @@
 #include "src/pltsql.h"
 #include "tcop/tcopprot.h"
 
-int			pgtsql_base_yydebug;
```

Comment:
These changes don't look relevant to the SSL changes. Are these needed? Is it causing build failure? Because our github actions are not showing any compilation errors.

Reply:
Can you put those in a separate pull request? We squash and merge PRs so these changes would also get bundled up in a single commit in upstream.